### PR TITLE
UX testing changes for Setup, What is Flutter?

### PR DESCRIPTION
**Pull request by LarkAscending**
Please review and approve ASAP - needed for First Look.

@lukechurch @InMatrix

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,26 +4,24 @@ sidebar: home_sidebar
...
 ![Shrine, iOS left and Android right](/images/whatisflutter/shrine_both.png)
 
-This is an online marketplace app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo),
-a showcase of Flutter apps to download and run locally. Shrine has
-high-quality scrolling images, interactive cards, buttons, dropdown lists,
-and a shopping cart page. You can run Shrine on an emulator or a phone, then check
-out the single codebase for this and more examples [here](https://github.com/flutter/flutter/tree/master/examples)
-in our GitHub repository.
+This is a demo app from the [Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery/lib/demo),
+a collection of Flutter sample apps you can run after installing Flutter and setting
```
Comment (mit-mit) [inline index.md]: The term 'Flutter sample apps' is a little confusing; this is a single iOS/Android app

**Comment by sethladd** [comment]
Thanks! Have we looked at using a JPEG for the image on the homepage? Or, using a tool like optipng? The current PNG is about half a meg, which is a lot for a homepage. Something we might want to look into for a future PR.

File: setup.md [hunk 2]
```diff
@@ -14,27 +14,27 @@ The remaining tasks can be deferred until later in the development cycle, when y
...
-  <td align="center">&#10004;</td><td><a href="#system-requirements">Verify your system meets the requirements for Flutter and dependencies</a></td><td align="center">Y</td>
+  <td style="width:1600px;"><a href="#system-requirements">Verify your system meets the requirements for Flutter and dependencies</a></td><td align="center">Y</td>
  </tr>
  <tr>
-  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
+  <td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
  </tr>
  <tr>
-  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
+  <td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
```
Comment (mit-mit) [inline setup.md]: Wrong href?

File: setup.md [hunk 3]
```diff
@@ -14,27 +14,27 @@ The remaining tasks can be deferred until later in the development cycle, when y
...
  </tr>
  <tr>
-   <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Run <code>flutter doctor</code> and resolve any issues found</a></td><td align="center">Y</td>
+   <td><a href="#get-the-flutter-sdk">Run <code>flutter doctor</code> and resolve any issues found</a></td><td align="center">Y</td>
```
Comment (mit-mit) [inline setup.md]: Wrong href?

File: setup.md (hunk 2 above)
Comment (LarkAscending) [inline setup.md]: As intended. This is a safe way to override a site-wide CSS style that makes the first td of every table 150px wide - very narrow indeed. It's needed to make this table look normal for First Look. 

After First Look, we'll diagnose and fix main.css.

File: setup.md (hunk 3 above)
Comment (LarkAscending) [inline setup.md]: Removed the first column of the table, per @lukechurch UX feedback.

File: index.md (hunk 1 above)
Comment (LarkAscending) [inline index.md]: The Gallery is an app w/many nested apps. It's both a single app and a collection of sample apps! We're both right!
:-D

**Pull request merged**